BondageClub/Screens/Character/Title/Title.js

```diff
@@ -180,22 +184,25 @@ function TitleClick() {
 	// When the user exits
 	if ((MouseX >= 1815) && (MouseX < 1905) && (MouseY >= 75) && (MouseY < 165)) TitleExit();
 
+	if (TitleListFiltered.length > TitlePerPage && MouseIn(1705, 75, 90, 90)) {
+		TitleOffset += TitlePerPage;
+		if (TitleOffset >= TitleListFiltered.length) TitleOffset = 0;
+	}
+
 	// When the user selects a title
-	var X = 130;
-	var Y = 250;
-	for (let T = 0; T < TitleList.length; T++)
-		if (TitleList[T].Requirement()) {
-			if ((MouseX >= X) && (MouseX <= X + 400) && (MouseY >= Y) && (MouseY <= Y + 65)) {
-				TitleSet(TitleList[T].Name);
-				TitleExit();
-			}
-			X = X + 450;
-			if (X > 1500) {
-				X = 130;
-				Y = Y + 90;
-			}
+	let X = 130;
+	let Y = 250;
+	for (let T = TitleOffset; T < TitleOffset + TitlePerPage && T < TitleListFiltered.length; T++) {
+		if ((MouseX >= X) && (MouseX <= X + 400) && (MouseY >= Y) && (MouseY <= Y + 65)) {
```

Comment:
Probably worth refactoring this to use `MouseIn` while we're making changes here.
```suggestion
		if (MouseIn(X, Y, 400, 65)) {
```

Reply:
sure, fixed the exit button as well then

Reply:
sure, fixed the exit button as well